app/views/declaration/officeOfExit/office_of_exit_standard.scala.html

```diff
@@ -14,60 +14,108 @@
+        )),
+        fieldset = Some(Fieldset(
+            legend = Some(Legend(
+                content = Text(messages("standard.officeOfExit.circumstancesCode")),
+                classes= "govuk-fieldset__legend--m",
+                isPageHeading = false
+            ))
+        )),
+        items = Seq(
+            RadioItem(
+                id = Some("Yes"),
+                value = Some(AllowedCircumstancesCodeAnswers.yes),
+                content = Text(messages("site.yes")),
+                checked = form("circumstancesCode").value.contains("Yes")
+            ),
+            RadioItem(
+                id = Some("No"),
+                value = Some(AllowedCircumstancesCodeAnswers.no),
+                content = Text(messages("site.no")),
+                checked = form("circumstancesCode").value.contains("No")
+            )
+        ),
+        errorMessage = form("circumstancesCode").error.map(err => ErrorMessage(content = Text(messages(err.message, err.args:_*))))
+    ))
+}
 
-        @components.heading(
-            sectionHeader = messages("declaration.summary.locations.header"),
-            title = messages("declaration.officeOfExit.title"))
+@govukLayout(
+    title = Title("declaration.officeOfExit.title", "declaration.summary.locations.header"),
+    backButton = Some(BackButton(messages("site.back"), Navigator.backLink(OfficeOfExitStandard, mode)))){
 
-        @components.fields.field_autocomplete(
-            field = form("officeId"),
-            label = messages("declaration.officeOfExit"),
-            labelClass = Some("form-label-bold"),
-            hintText = Some(messages("declaration.officeOfExit.hint")),
-            messages("declaration.officeOfExit.empty"),
-            AutoCompleteItem.fromOfficeOfExit(OfficeOfExits.all)
-        )
+    @formHelper(action = controllers.declaration.routes.OfficeOfExitController.saveOffice(mode), 'autoComplete -> "off") {
+        @errorSummary(form.errors)
 
-        @components.fields.field_radio(
-            field = form("circumstancesCode"),
-            legend = messages("standard.officeOfExit.circumstancesCode"),
-            hint = Some(messages("standard.officeOfExit.circumstancesCode.hint")),
-            inputs = Seq(
-                RadioOption("Yes", AllowedCircumstancesCodeAnswers.yes, messages("site.yes")),
-                RadioOption("No", AllowedCircumstancesCodeAnswers.no, messages("site.no"))
-            )
-        )
+        @sectionHeader(messages("declaration.summary.locations.header"))
 
-        @components.details(messages("site.details.summary_text_these")){
-           @tariffBody
-        }
-        @components.submit_button()
-        @components.buttons.save_and_return_later_button()
+        @govukFieldset(Fieldset(
+            legend = Some(Legend(
+                content = Text(messages("declaration.officeOfExit.title")),
+                classes = "govuk-fieldset__legend--xl",
```

Comment:
Can you use the constant Styles.gdsPageLegend here for the classes.

Reply:
Fixed!